File: Producto2/Hola.cpp
#include <iostream>
#include <time.h>
using namespace std;

int main()
{
   cout << "Hola! Trataré de adivinar un número" << endl; 
   cout << "Piensa un número entre 1 y 10" << endl; 
   sleep(5);
   cout << "Ahora multiplícalo por 9." << endl; 
   sleep(5);
   cout << "Si el número tiene 2 dígitos, súmalos entre si. Si tu número tiene un solo dígito, súmale 0." << endl; 
   sleep(5);
   cout << "Al número resultante súmale 4" << endl; 
   sleep(10);
   cout << "Muy bien. El resultado es 13:D" << endl; 
   
   return 0;
}

File: Producto2/Hola.java
public class HelloWorld{

     public static void main(String []args){
       
        System.out.println("Hola! tratare de adivinar un numero.");
        try {Thread.sleep(5000); }
        catch (Exception e) { }
        System.out.println("Piensa un numero entre 1 y 10.");
        try {Thread.sleep(5000); }
        catch (Exception e) { }
        System.out.println("Ahora multiplicalo por 9.");
        try {Thread.sleep(5000); }
        catch (Exception e) { }
        System.out.println("Si el numero tiene 2 digitos, sumalos entre si. Si tu numero tiene un solo digito, sumale 0.");
        try {Thread.sleep(5000); }
        catch (Exception e) { }
        System.out.println("Al numero resultante sumale 4.");
        try {Thread.sleep(10000); }
        catch (Exception e) { }
        System.out.println("Muy bien. El resultado es 13:D");
         
     }
}